client/views/flashcards/Flashcard.js

```diff
@@ -0,0 +1,18 @@
+Template.Flashcard.events({
+  "click .flip": function(event, template){
+    template.$('.front').toggleClass('hidden');
+    template.$('.back').toggleClass('hidden');
+  }
+});
+
+Template.Flashcard.helpers({
+  indexIsZero: function() {
+    return this.index == 0
+  }
+});
+
+UI.registerHelper('addIndex', function (all) {
```

Comment:
Yeah its safe to remove, it was an original idea I had that got merged into my work. I forgot to remove it.